localisation/00_ciwc_united_localisation.csv

```diff
@@ -17,4 +17,21 @@ great_sea_trade_route_tooltip;Can only build 
 ;;;;;;;;;;;;;;x
 text_effect_major_trade_route;This is �GMajor�! Trade Route;;;;;;;;;;;;;x
 text_effect_medium_trade_route;This is �YMedium�! Trade Route;;;;;;;;;;;;;x
-text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x
\ No newline at end of file
+text_effect_minor_trade_route;This is �RMinor�! Trade Route;;;;;;;;;;;;;x

```

Comment:
Shouldn't it be "This is **a** Minor/Medium/Major Trade Route?"

Reply:
Oh this isjt from my code this is from base game i believe

Reply:
could be, but who are we **not** to improve vanilla game where we find problems/mistakes?^^

Reply:
First, it's my code from previous trade branch.
Second, it's not part of this branch, if you want to change it - do it on branch for localization improvements,

Reply:
@arithon #951

Reply:
I think ive said before but this is not part of my mod but something that was there before ui believe from a previous pr but i can fix that in this one

Reply:
No need. Accounted for in ca70621.